frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx

```diff
@@ -457,7 +457,7 @@ export const HWCRComplaintAssessment: FC<Props> = ({
       (!selectedAssessmentTypes || selectedAssessmentTypes?.length <= 0) &&
       (!selectedAssessmentCat1Types || selectedAssessmentCat1Types?.length <= 0)
     ) {
-      setAssessmentRequiredErrorMessage("One or more assessment is required");
+      setAssessmentRequiredErrorMessage("One or more animal actions is required");
```

Comment:
Not a technical issue - the grammar here is a bit weird. I'd suggest 'One or more animal actions are required'